src/db.py

```diff
@@ -16,6 +17,13 @@ def is_valid(cls, quota):
 		return quota in [cls.DAILY, cls.TOTAL]
 
 
+# Assignment visibility to students
+class Visibility:
+	HIDDEN = "hidden"
```

Comment:
Can we use an enum for this?
You can make this class extend `Enum`, so that they are all increasing numbers.

Reply:
Agree

Reply:
Because Mongodb doesn't support directly storing enum, changing this to Enum will add an extra ".value" on every usage. I don't think that's necessarily better. Examples:
```                                                    
{% if assignment.visibility == visibility.HIDDEN.value %}
  checked
{% endif %}
```

Reply:
I can manually change the strings to integers if you're concerned about storage space. But storing strings are more explicit if someone hasn't looked at the code but go straight to the db.

Reply:
Hmm fair enough we can stick to strings.